src/python/bcc/table.py

```diff
@@ -186,6 +186,10 @@ def clear(self):
         for k in self.keys():
             self.__delitem__(k)
 
+    def zero(self):
+        for k in self.keys():
+            self[k] = self.Leaf()
```

Comment:
I don't think this part actually clear it in a hash map.
Isn't zero-ing it in python memory only?